scripts/idl/generators/bridge/BridgeClustersCpp.jinja

```diff
@@ -59,6 +59,20 @@ struct {{cluster.name}}Cluster : public GeneratedCluster
     });
   }
 
+  {%- if cluster.commands | length > 0 %}
+  chip::CommandId incomingCommandList[{{cluster.commands | length + 1}}] = {
+  {%- for command in cluster.commands %}
+    {{command.code}},
+  {%- endfor %}
+    chip::kInvalidCommandId
+  };
+
+  chip::CommandId * GetIncomingCommandList() override
```

Comment:
Is there a unit test that exercises this new code generation?
I don't see one - If I missed it, please let me know where it is. If one does not exist, please create one: I want to see what code gets generated and we want to see any changes to generated code over time.

Reply:
Added a test `cluster_with_commands` in `scripts/idl/tests`; Input is `scripts/idl/inputs/cluster_with_commands.matter`, and output is `scripts/idl/outputs/cluster_with_commands/bridge/`.

See [b13861c](https://github.com/project-chip/connectedhomeip/pull/23840/commits/b13861cfd7130f40d06ac24b54c5decda897eaaf)